microsoft-edge/devtools-guide-chromium/experimental-features/index.md

```diff
@@ -811,77 +855,77 @@ See also:
 * [Record heap snapshots using the Memory tool](../memory-problems/heap-snapshots.md)
 
 Status:
-* This checkbox is present in Microsoft Edge Canary 125.
-* This checkbox is present in Microsoft Edge Stable 124.
+* This checkbox is present in Microsoft Edge Canary 128.
+* This checkbox is present in Microsoft Edge Stable 125.
 
 
 <!-- ====================================================================== -->
-## Timeline: invalidation tracking
-<!-- checkbox has a (?) link to this anchor wording -->
+## Performance panel: invalidation tracking
+<!-- checkbox has no (?) link pointing to this anchor wording -->
 
 Controls whether to show invalidation tracking in the **Performance** tool.
+<!-- todo: real desc for this older entry -->
```

Comment:
Performance panel: invalidation tracking
Need real desc @captainbrosset

Reply:
Adds an **Invalidations** section for **Recalculate Style** events under the **Summary** tab of the **Performance** tool. The **Invalidations** section provides information about the root cause for a **Recalculate Style** event, which might be helpful when investigating long **Recalculate Style** events that are causing performance issues.

The **Invalidations** section lists the DOM nodes that got invalidated as a result of a change on the page, such as a DOM mutation, and which the browser engine had to restyle during the **Recalculate Style** event.

Reply:
```
Adds an **Invalidations** section for **Recalculate Style** events in the **Summary** tab 
of the **Performance** tool.  The **Invalidations** section provides information about 
the root cause for a **Recalculate Style** event, which might be helpful when 
investigating long-running **Recalculate Style** events that are causing performance 
issues.

The **Invalidations** section lists the DOM nodes that were invalidated as a result of 
a change on the webpage, such as a DOM mutation, and which the browser engine 
had to restyle during the **Recalculate Style** event.
```